src/Simplify_Stmts.cpp

```diff
@@ -501,5 +501,22 @@ Stmt Simplify::visit(const Fork *op) {
     }
 }
 
+Stmt Simplify::visit(const Atomic *op) {
+    bool changed = false;
```

Comment:
Don't need the changed variable, as in the simplification to IRMutator.cpp

Reply:
Done.